test/catch.hpp

```diff
@@ -10291,8 +10291,8 @@ namespace Catch {
 
 #if defined(CATCH_PLATFORM_MAC) || defined(CATCH_PLATFORM_IPHONE)
 
-#  include <assert.h>
-#  include <stdbool.h>
+#  include <cassert>
```

Comment:
This header is take from https://github.com/catchorg/Catch2 - please update it there.